pset-iterators.js

```diff
@@ -66,6 +72,14 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+const getFullName = presidents.map((objectPresident) => {
```

Comment:
This definitely works, but remember the .map method returns something for each iteration (so you may have to tell it to return nothing if James is not the first name). Another potential iteration solution that uses the power of conditionals is the .filter method, an example of that can be found [here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L66-L70).